REFERENCE.md

```diff
@@ -0,0 +1,247 @@
+# Reference
+<!-- DO NOT EDIT: This document was generated by Puppet Strings -->
+
+## Resource types
+
+* [`zfs`](#zfs): Manage zfs. Create destroy and set properties on zfs instances.  **Autorequires:** If Puppet is managing the zpool at the root of this zfs in
+* [`zpool`](#zpool): Manage zpools. Create and delete zpools. The provider WILL NOT SYNC, only report differences.  Supports vdevs with mirrors, raidz, logs and s
+
+## Resource types
```

Comment:
@jbondpdx any thoughts on why I'm getting `## Resource types` twice here? I can't figure out if it's something I did with the puppet strings documentation for these files, or it's something in how puppet strings itself is generating this file

Reply:
Hmmm, this one I don't know. @eputnam ?

Reply:
The first `## Resource types` is part of the table of contents, the second is denoting the `## Resource types` section

Reply:
If the first one is part of the ToC, then it probably shouldn't be an H2, like the section is? That, or there should be some clear indication of it being part of a ToC. As it is, the rendered output looks like it's broken:
![image](https://user-images.githubusercontent.com/14557/40331843-8f39be8a-5d06-11e8-9c88-bd981678bbce.png)

Reply:
D'oh!

Reply:
yeah, an oversight for sure. we were testing on larger modules.

Reply:
https://tickets.puppetlabs.com/browse/PDOC-255 covers the issue here. No reason to block this PR for something that'll be fixed in Puppet Strings.